src/python/setup.py

```diff
@@ -32,15 +30,27 @@
     author="The tensorflow cloud authors",
     author_email="tensorflow-cloud@google.com",
     license="Apache License 2.0",
-    install_requires=["docker", "google-api-python-client", "google-cloud-storage",],
-    extras_require={"tests": ["pytest", "flake8", "mock"],},
+    extras_require={"tests": dependencies.make_required_test_packages()},
+    include_package_data=True,
+    install_requires=dependencies.make_required_install_packages(),
     classifiers=[
-        "Programming Language :: Python",
+        "Development Status :: 4 - Beta",
         "Operating System :: Unix",
         "Operating System :: MacOS",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Education",
         "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
         "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Artificial Intelligence",
         "Topic :: Software Development",
+        "Topic :: Software Development :: Libraries",
+        "Topic :: Software Development :: Libraries :: Python Modules",
     ],
+    package_data={"tensorflow_cloud": ["tuner/api/*.json"]},
```

Comment:
Can you explain what this does?

Reply:
ml_public_google_rest_v1.json contains the API description and metadata. It is required for the Tuner package. Following [setupscript guidelines](https://docs.python.org/2/distutils/setupscript.html#installing-package-data) including this data in the .whl package explicitly.